test/api/v3/unit/models/group.test.js

```diff
@@ -366,7 +366,7 @@ describe('Group Model', () => {
 
           party = await Group.findOne({_id: party._id});
 
-          expect(Group.prototype.sendChat).to.be.calledOnce;
+          expect(Group.prototype.sendChat).to.be.calledTwice;
```

Comment:
Can we instead reset the spy after the `startQuest` call?

```js
await party.startQuest(questLeader);
Group.prototype.sendChat.reset();
```

That will put the spy in a clean state for the assertion we're actually looking to make here.